new/PRS12787GabraEgziabeher.xml

```diff
@@ -48,18 +50,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <body>
             <listPerson>
                 <person sex="1">
-                    <persName xml:lang="gez" xml:id="n1">GabraEgziabeher</persName>
-                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
+                    <persName xml:lang="gez" xml:id="n1">ገብረ᎓ እግዚአብሔር᎓</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Gabra ʾƎgziʾabǝḥer</persName>
                     <faith type="EOTC"/>
                     <occupation type="other"/>
                     <birth/>
                     <death/>
                     <floruit/>
                 </person>
                 <listRelation>
-                    <relation name="saws:isRelatedTo" active="PRS12787GabraEgziabeher" passive=""/>
+                    <relation name="saws:isRelatedTo" active="PRS12787GabraEgziabeher" passive="RNBefsn7"/>
```

Comment:
maybe better isAttestedIn

Reply:
well, he is rather (actively)  related then mentioned. For me, a person who is mentioned in the text as protagonists would be "mentioned".